### ISSUE TITLE
Human Arms Turn Invisible After Reloading Save

### ISSUE THREAD

**Issue by genessee-forgot-his-password-again** [PersistentSS13/Nebula#413]
#### Description of issue

People's arms become invisible after loading them in a save. They still work as though they had arms, but it looks really weird.

![image](https://user-images.githubusercontent.com/80235421/222055283-e6affd0b-513d-4e15-a9ae-897efd76a562.png)

#### Difference between expected and actual behavior

I expected people who have arms to look like they have arms. Instead, they have weird Phantom Limb from Venture Brothers arms.

#### Steps to reproduce

1. Spawn as a colonist.
2. Save.
3. Load save.
4. Observe as your arms are now invisible.

#### Specific information for locating

Genuinely no idea what's causing it. I couldn't give you any atompaths or errors.

#### Length of time in which bug has been known to occur

It was occuring during the last open test a year ago, but I forgot to make an issue. Cropped up during testing again and I decided to make an issue now.

#### Client version, Server revision & Game ID

![image](https://user-images.githubusercontent.com/80235421/222055511-73976b85-1729-4b01-951e-66147fd6b89f.png)

#### Issue bingo

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/NebulaSS13/Nebula/issues)

**Comment by PsyCommando** [comment]
Yeah, that's a known issue. It's due to the way human mobs initialize. It skips over calling some of the icon update code.
